go.mod

```diff
@@ -291,7 +291,7 @@ require (
 
 replace (
 	// TODO MUST BE REMOVED IN THE NEXT FOLLOWING PR (or this PR needs to be split and cosmossdk.io/api tagged)
-	cosmossdk.io/api => ./api
+	cosmossdk.io/api => cosmossdk.io/api v0.1.0-alpha10
```

Comment:
I think I need to tag orm, core and tx first and update them after this PR. And then to have a last PR for removing this?